io-ballerina/src/io/bytes/writable_byte_channel.bal

```diff
@@ -15,28 +15,32 @@
 // under the License.
 import ballerina/java;
 
-# WritableByteChannel represents an output resource (i.e file). which could be used to sink bytes.
+# WritableByteChannel represents an output resource (i.e file). which could be used to sink bytes
+# A file path can be used to obtain a WritableByteChannel.
+# A WritableByteChannel do not support initilization, and it should be obtained using the following method or implement natively.
+# `io:openWritableFile("./files/sample.txt")` - used to obtain a WritableByteChannel from a given file path
 public class WritableByteChannel {
 
-    # Adding default init function to prevent object getting initialized from the user code.
+    # Adding default init function to prevent object getting initialized from the user code
     function init() {
     }
 
-    # Sinks bytes from a given input/output resource.
+    # Sinks bytes from a given input/output resource
     #
     # This operation will be asynchronous. Writing might return without writing all the content.
```

Comment:
```suggestion
    # This is an asynchronous operation. The method might return before writing all the content.
```

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-io/pull/37/commits/123095180ab4fe7865b5e69d8623895fe6b978e8